src/main.ts

```diff
@@ -99,61 +104,21 @@ export class Minifier {
   private checkReserved(str: string): boolean {
     // From MDN's Lexical Grammar page
     // (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar)
-    var reserved = [
-      'break',
-      'case',
-      'class',
-      'catch',
-      'const',
-      'continue',
-      'debugger',
-      'delete',
-      'do',
-      'else',
-      'export',
-      'extends',
-      'finally',
-      'for',
-      'function',
-      'if',
-      'import',
-      'in',
-      'instanceof',
-      'let',
-      'new',
-      'return',
-      'super',
-      'switch',
-      'this',
-      'throw',
-      'try',
-      'typeof',
-      'var',
-      'void',
-      'while',
-      'with',
-      'yield',
-      'enum',
-      'await',
-      'int',
-      'byte',
-      'char',
-      'goto',
-      'long',
-      'final',
-      'float',
-      'short',
-      'double',
-      'native',
-      'throws',
-      'boolean',
-      'abstract',
-      'volatile',
-      'transient',
-      'synchronized'
-    ];
-
-    return (reserved.indexOf(str) > -1);
+    var keywordList =
```

Comment:
You're doing this work on every call to this method. That's not a good idea, is it?